enhancements/network/ovn-kubernetes-multiple-external-gateway.md

```diff
@@ -0,0 +1,180 @@
+          * `skipHostSNAT`: When true, it disables applying the Source NAT to the host IP.
+          * `bfdEnabled`: determines if Bi-Directional Forwarding Detection (BFD) is supported in this network and follows the same behavior as with the annotation: it's enabled when set to true and disabled when false or omited in the manifest. It maps to the `k8s.ovn.org/bfd-enabled` annotation in the pod. This field is optional.
+          * `networkAttachmentName`: Name of the network attached definition. The name needs to match from the list of logical networks generated by Multus and associated to the pod via annotation. This field is optional. When this field is empty or not defined, the IP from the pod's host network will be used instead. The pod must have `hostnetwork` enabled in order for this feature to work.
+
+The following yaml examples an instance of the CRD:
+
+```yaml
+apiVersion: k8s.ovn.org/v1
+kind: AdminPolicyBasedExternalRoute
+metadata:
+  name: honeypotting
+spec:
+  policies:
+  ## gateway example
+  - from:
+      namespaceSelector:
+        matchLabels:
+            multiple_gws: true
+    nexthops:
+      static:
+        - ip: "172.18.0.2"
+          bfdEnabled: true
+        - ip: "10.244.0.3"
+          skipHostSNAT: true
+      dynamic:
+        - podSelector:
+            matchLabels:
+              external-gateway: true
+          bfdEnabled: true
+          namespaceSelector:
+            matchLabels:
+              gateway: true
+          networkAttachmentName: sriov1
+        - podSelector:
+            matchLabels:
+              secondary-gateway: true
+          skipHostSNAT: true
+          namespaceSelector:
+            matchLabels:
+              secondary-gateway: true
+          networkAttachmentName: sriov2          
+```
+
+### Risks and Mitigations
+N/A
+
+### Drawbacks
+N/A
+
+## Design Details
+
+
+### Test Plan
+
+* Unit test coverage for CR based to at least match use cases in existing annotation based test cases.
+* Unit test for both annotation and CR to include tests cases to validate expected behavior when running in OpenShift 4.13. 
+
+### Graduation Criteria
+N/A
+
+#### Dev Preview -> Tech Preview
+This feature is GA in 4.13, no Dev or Tech previews are required because the logic already exists in OVN-K as an annotation.  
+
+
+#### Tech Preview -> GA
+This feature is GA in 4.13, no Dev or Tech previews are required because the logic already exists in OVN-K as an annotation.
+
+
+#### Removing a deprecated feature
+Removal of the annotation based logic to occur in OpenShift 4.14.
+
+### Upgrade / Downgrade Strategy
+There are no plans, tools or documentation to assist in migrating the annotation based solution to the CRD based approach.
+
+### Version Skew Strategy
+N/A
+
+### Operational Aspects of API Extensions
+N/A
+
+#### Failure Modes
+N/A
+
+#### Support Procedures
+N/A
+## Implementation History
+First implementation targeting OpenShift 4.13 in GA capacity.
```

Comment:
nit: probably good to add `upstream docs` for this feature in its goals (I know downstream docs are a must for an API feature).. this can help other engineers for knowledge sharing.